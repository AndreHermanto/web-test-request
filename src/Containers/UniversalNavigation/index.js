import React from 'react';
import { Nav, Navbar,  NavItem } from 'react-bootstrap';
export default class UniversalNavigation extends React.Component 
{    
  render() 
  {  
    return( 
      <Navbar>
        <Navbar.Header>
          <Navbar.Brand>
            <a href="#">Home</a>
          </Navbar.Brand>
        </Navbar.Header>
        <Nav>
          <NavItem eventKey={1} href="#">Link</NavItem>
          <NavItem eventKey={2} href="#">Link</NavItem>
        </Nav>
      </Navbar>
    )
  }
}
