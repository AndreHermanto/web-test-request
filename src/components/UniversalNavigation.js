import React from 'react';
import { Navbar, Grid } from 'react-bootstrap';
import styled from 'styled-components';
import logo from '../assets/images/g1_logo.png';

const Bar = styled(Navbar)`
  background-image: none !important;
  background-repeat: no-repeat !important;
`;

const NavTitle = styled.span`
  margin-left: 32px;
`;

export default class UniversalNavigation extends React.Component 
{    
  render() 
  {  
    return( 
      <Bar>
        <Grid>
          <Navbar.Header>
            <Navbar.Brand>
              <a href="/">
                <img 
                  src={logo} 
                  alt="logo" 
                  style={{ width: 80, height:30, marginTop: -4 }} />
              </a>
              <NavTitle>Test Request App</NavTitle>
            </Navbar.Brand>
            <Navbar.Toggle />
          </Navbar.Header>
        </Grid>
      </Bar>
    )
  }
}
