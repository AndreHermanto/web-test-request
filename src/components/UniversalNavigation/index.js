import React from 'react';
import { Nav, Navbar,  NavItem } from 'react-bootstrap';

function UniversalNavigation()
{     
  return( 
    <Navbar>
      <Navbar.Header>
        <Navbar.Brand>
          <a href="#">Home</a>
        </Navbar.Brand>
      </Navbar.Header>
      <Nav>
        <NavItem eventKey={1} href="#">Link</NavItem>
        <NavItem eventKey={2} href="#">Link</NavItem>
      </Nav>
    </Navbar>
  )
}

export default UniversalNavigation;
