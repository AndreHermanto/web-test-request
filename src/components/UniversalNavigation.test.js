import React from 'react';
import UniversalNavigation from './UniversalNavigation';
import renderer from 'react-test-renderer';

describe('<UniversalNavigation />', function() {
  it('renders correctly', function() {
    const tree = renderer.create(<UniversalNavigation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});