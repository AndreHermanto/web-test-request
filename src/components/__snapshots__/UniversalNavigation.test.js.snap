exports[`<UniversalNavigation /> renders correctly 1`] = `
<nav
  className="fvulNi navbar navbar-default">
  <div
    className="container">
    <div
      className="container"
      style={
        Object {
          "maxWidth": 940,
          "padding": 0,
        }
      }>
      <div
        className="navbar-header">
        <span
          className="navbar-brand">
          <a
            href="/">
            <img
              alt="logo"
              src="test-file-stub"
              style={
                Object {
                  "height": 64,
                  "marginTop": -8,
                  "width": "auto",
                }
              } />
          </a>
          <span
            className="hdMxID">
            Test Request App
          </span>
        </span>
        <button
          className="navbar-toggle collapsed"
          onClick={[Function]}
          type="button">
          <span
            className="sr-only">
            Toggle navigation
          </span>
          <span
            className="icon-bar" />
          <span
            className="icon-bar" />
          <span
            className="icon-bar" />
        </button>
      </div>
    </div>
  </div>
</nav>
`;
